src/connector/src/parser/mod.rs

```diff
@@ -974,6 +974,7 @@ pub enum EncodingProperties {
     Json(JsonProperties),
     Bytes(BytesProperties),
     Native,
+    None,
```

Comment:
`None` seems a bit meanlingless 🤔

```suggestion
    /// Encoding can't be specified because the source will determines it. Now only used in Iceberg. 
    None,
```